vendor/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/operators/machine-api-operator.go

```diff
@@ -23,7 +23,7 @@ var _ = g.Describe("[Feature:Operators] Machine API operator deployment should",
 		client, err := e2e.LoadClient()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		deploymentName := "clusterapi-manager-controllers"
+		deploymentName := "machine-api-controllers"
```

Comment:
Is there a PR for this on cluster-api-actuator-pkg?

Reply:
https://github.com/openshift/cluster-api-actuator-pkg/pull/63